cmd/fileRefs.go

```diff
@@ -0,0 +1,109 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/0chain/gosdk/zboxcore/fileref"
+	"github.com/0chain/gosdk/zboxcore/sdk"
+	"github.com/0chain/zboxcli/util"
+	"github.com/spf13/cobra"
+)
+
+const (
+	PageLimit = 100
+	Layout    = "2006-01-02 15:04:05"
+)
+
+func checkError(err error) {
+	if err != nil {
+		PrintError(err)
+		os.Exit(1)
+	}
+}
+
+var fileRefsCmd = &cobra.Command{
+	Use:   "recent-refs",
+	Short: "get list of recently added refs",
+	Long:  `get list of recently added refs`,
+	Args:  cobra.MinimumNArgs(0),
+	Run: func(cmd *cobra.Command, args []string) {
+		page, err := cmd.Flags().GetUint("page")
+		checkError(err)
+		pageLimit, err := cmd.Flags().GetUint("page_limit")
+		checkError(err)
+		if pageLimit > PageLimit {
+			PrintError("Invalid page limit value. Should be in range (0,100]")
+			os.Exit(1)
+		}
+		allocID, err := cmd.Flags().GetString("allocation")
+		checkError(err)
+		fromDate, err := cmd.Flags().GetDuration("from_date")
```

Comment:
Hmm, seems the from_date is not a date base on the code below where it is used. It is more like a last `N` days or minutes. The `date` may may confuse users, perhaps use `in`, so the command could be used like `recent-refs -in 3d` ? or use it as a date, so the code in line 56 below would not needed, but the date format should be defined.